### ISSUE TITLE
It's hard to find a particular job amidst multiple jobs

### ISSUE THREAD

**Issue by marisademeglio** [daisy/pipeline-ui#14]
"Open Pipeline, select any number of scripts and run it. If I have multiple jobs open and I want to go to any particular job then I have to pass through all the jobs."

cc @rachanasinghg

**Comment by marisademeglio** [comment]
@rachanasinghg is it hard to find a particular job because of side-scrolling caused by #13 ? Or is it something else that causes difficulty?

**Comment by rachanasinghg** [comment]
Hi Marisa,

It is somewhat similar to the scrolling part but not the same. If there are
3 jobs and i want to go to middle one ( eg. Job 2) then focus can go to the
close button but i can't move inside it.
I am attaching the image for reference.

With Regards
Rachana Singh
Usability & Testing Engineer | DAISY Consortium
<https://daisy.org/activities/software/obi/>

**Comment by marisademeglio** [comment]
So you're talking about keyboard navigation to locate a tab, not mouse pointer? 

Could you attach the image again? I'm not seeing it here. Thanks!

**Comment by rachanasinghg** [comment]
Yes, i am talking about keyboard navigation to locate a tab, with a mouse
pointer it's easy to select any job.
Sending the image again.
Thanks.

With Regards
Rachana Singh
Usability & Testing Engineer | DAISY Consortium
<https://daisy.org/activities/software/obi/>

**Comment by marisademeglio** [comment]
Ok thanks!  So if we call the top part the "tabs" and the main part the "tab panels" -- 

So you want to be focused on the tab and then have a fast way to move to the contents of its corresponding tab panel?

**Comment by rachanasinghg** [comment]
@DeMeglio Marisa ***@***.***> Yes, right I want a way to
move inside the contents of focussed tab. With keyboard navigation it just
focuses on the close button.
Thanks for understanding.

With Regards
Rachana Singh
Usability & Testing Engineer | DAISY Consortium
<https://daisy.org/activities/software/obi/>

**Comment by prashantverma2014** [comment]
The issue persists. Try these steps with keyboard only without screen reader.
1. Create several jobs by pressing Enter on New Job
2. Press tab and you can move to the script selection within the selected tab  say 4. New job. 
3. Now try to go back to another job tab. tried using Tab, Shift tab, arrow keys but cannot move to another job tab e.g. "3. New job"

**Comment by marisademeglio** [comment]
This should be resolved in https://github.com/daisy/pipeline-ui/commit/0e57c9932fd75d64c24f914c2107f072498152f4

* `Ctrl + Tab` and `Ctrl + Shift + Tab` move foward and back through the jobs list
* There are also menu commands for Next/Prev job.
